spec/system/candidate_interface/carry_over/candidate_carries_over_unsubmitted_application_spec.rb

```diff
@@ -4,12 +4,11 @@
   include CandidateHelper
 
   scenario 'Candidate carries over their application to the new cycle' do
-    given_i_am_signed_in
+    given_i_am_signed_in_with_one_login
     and_i_have_an_unsubmitted_application_with_references
 
     when_the_apply_deadline_passes
     and_i_sign_in_again
-    and_i_visit_the_application_dashboard
```

Comment:
Are we using this method anywhere now that it's been deleted from this test? If not, can we delete it from the test file so we don't have unused code?